deployment/conda/conda_env.yml

```diff
@@ -32,9 +35,11 @@ dependencies:
 - geojson
 - flake8
 - pytest
-# Rutgers needs 0.4.*, we need cuda 9 and python 3.6
-- cuda90
-- pytorch=0.4.*
+- cuda100
+- cudatoolkit=10.0
```

Comment:
I'm not certain, but I believe these may be redundant I think the difference in name could be attributed to what conda channel it's coming from.  I've usually seen `cudatoolkit` used, but it looks like `cuda90` was used previously.  In either case, if *possible* let's try and stick with just one.

Reply:
As discussed on the call today, please create an issue mentioning this as something we should look into (as well as potentially upgrading the `pytorch` version).  Resolving as this should be resolved with a subsequent PR.